FORK.md

```diff
@@ -47,11 +49,35 @@ PRs must be created against the `fork` branch.
 
 To help keep things trackable, the following branch naming conventions
 are used:
+
 - New feature: `[username]/feature-[feature-name]`
 - Bug fix: `[username]/fix-[fix-name]`
 
 ## Changelog
 
+- Add support for rendering additional metadata
+    - Upstream issue: https://github.com/dbt-labs/dbt-core/issues/7021
```

Comment:
I believe it should be https://github.com/dbt-labs/dbt-docs/issues/424, right?

Reply:
Yes, it should! Thanks